packages/wonder-blocks-tokens/src/tokens/media-queries.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * A default set of media queries to use for different screen sizes.
+ *
+ * Breakpoint documentation: https://khanacademy.atlassian.net/wiki/spaces/WB/pages/2099970518/Layout+Breakpoints
+ *
+ */
+
+const width = {
+    xsMax: 567,
+    smMin: 568,
+    smMax: 681,
+    mdMin: 682,
+    mdMax: 1023,
+    lgMin: 1024,
+} as const;
+
+const mediaQuery = {
+    // Named
+    xs: `@media screen and (max-width: ${width.xsMax}px) /* breakpoint.mediaQuery.xs */`,
```

Comment:
Is `/* breakpoint.mediaQuery.xs */` useful for IDE auto completion or something else?

I was looking at the Docs and that info appears duplicated in the table.

<img width="1086" alt="Screenshot 2024-11-04 at 12 02 33 PM" src="https://github.com/user-attachments/assets/a8900d09-eb49-4e29-86bf-384530c0e591">

Again, if this is helpful for DX reasons, you can surely leave it as it is.

Reply:
Good question! It was a trick to avoid conflicts for typing of strings. I included it based on this comment in webapp from @kevinb-khan: https://github.com/Khan/webapp/blob/55c4465017a829d53bdc6d00d94d6d93e3b5dfe4/services/static/javascript/shared-styles-package/media-queries.ts#L26

> We attach a unique comment string to each of the media queries in this file. That way, if another media query has the same constraints as one of these queries, their strings won't be identical, and they therefore won't conflict.

Admittedly, the typing for this was really hard to get right and I still wasn't that happy with it in the end. It would be one of the first things I'd want to refactor for the Aphrodite successor.